tests/tests/test_guest_fail.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Canonical Ltd.
+# Authors:
+# - Hector Cao <hector.cao@canonical.com>
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+import subprocess
+import time
+import re
+
+import Qemu
+import util
+
+script_path=os.path.dirname(os.path.realpath(__file__))
+
+def test_guest_noept_fail():
+    """
+    tdx_NOEPT test case (See https://github.com/intel/tdx/wiki/Tests)
+    """
+
+    # Get initial dmesg contents for comparison later
+    cs = subprocess.run(['sudo', 'dmesg'], check=True, capture_output=True)
+    assert cs.returncode == 0, 'Failed getting dmesg'
+    dmesg_start = str(cs.stdout)
+
+    # Setup ept=0 in driver
+    cs = subprocess.run(['sudo', 'rmmod', 'kvm_intel'], check=True)
```

Comment:
the rmmod will fail if there is any VM running, `rmmod: ERROR: Module kvm_intel is in use` ; i dont know how we can  handle this situation